src/data_gradients/datasets/__init__.py

```diff
@@ -8,5 +13,7 @@
     "YoloFormatDetectionDataset",
     "VOCSegmentationDataset",
     "VOCFormatSegmentationDataset",
+    "CocoSegmentationDataset",
```

Comment:
Coco or COCO?

Reply:
Changed in [4f0c550](https://github.com/Deci-AI/data-gradients/pull/130/commits/4f0c5503f7627f5bb1da8b171675e19ced5c588b)